### DISCUSSION TITLE
如何加入专业术语词，来优化纠错

### DISCUSSION THREAD

**Discussion by Azhuang-git** [shibing624/pycorrector#529] [Q&A]
在做特定领域的语音转文字工作，希望引入纠错模型来纠正口音问题，但是遇到了两个问题：
1. 模型根据形似来判断错误，在音频转文字当中，反而导致了误判，根据形似来判断错误的功能是否可以调整关闭或者弱化权重？
2. 专业术语的表达，会被修正为错误的表达，我应该怎么样形成一个恰当的白名单？

**Comment by Azhuang-git** [comment]
我可以在Corrector类实例化的时候，对于自定义词频、停用词等基本信息，通过修改这些配置文件，来适应自己的特殊场景吗？

**Reply by shibing624** [reply]
可以，也能加混淆集纠错方法。

**Reply by Azhuang-git** [reply]
谢谢您的答复！我在查看类实例的时候，发现“形似”好像也是单独的函数，我应该也可以通过屏蔽这部分功能，来实现只纠“音似”的功能

**Reply by XiaoX1993** [reply]
> 可以，也能加混淆集纠错方法。

这个混淆集方法是不是只能用kenlm模型，无法使用gpt模型？
